tests/vufind_templates.php-cs-fixer.php

```diff
@@ -41,10 +33,8 @@
     'no_leading_namespace_whitespace' => true,
     'no_mixed_echo_print' => true,
     'no_singleline_whitespace_before_semicolons' => true,
-    'no_spaces_after_function_name' => true,
     'no_spaces_around_offset' => true,
-    'no_spaces_inside_parenthesis' => true,
-    'no_trailing_whitespace' => true,
+    'no_trailing_whitespace_in_comment' => false,
```

Comment:
Why do we need this?

Reply:
If we have a comment inside a single line <?php tag, this removes the space between the comment and the ?>.